packages/components/src/Typography/Typography.tsx

```diff
@@ -51,18 +53,27 @@ export function Typography({
   textCase,
   textColor,
   emphasisType,
+  numberOfLines,
 }: TypographyProps) {
+  const shouldTruncateText = numberOfLines && numberOfLines > 0;
   const className = classnames(
     styles.base,
     fontWeights[fontWeight],
     size && fontSizes[size],
     textCase && textCases[textCase],
     textColor && textColors[textColor],
     emphasisType && emphasis[emphasisType],
+    shouldTruncateText && truncate.textTruncate,
   );
 
+  let truncateLines: CSSProperties | undefined;
+  if (shouldTruncateText) {
+    truncateLines = {};
+    truncateLines.WebkitLineClamp = numberOfLines;
+  }
```

Comment:
I wonder if this makes more sense to look something like this

```
const truncateLines: CSSProperties | undefined = {};
  if (shouldTruncateText) {
    truncateLines.WebkitLineClamp = numberOfLines;
  }
```

Since we are sending the var to the style attribute no matter what.

Reply:
That changed almost every single snapshot file. So this way if we don't numberOfLines then style is not added to the Tag and the snapshots don't fail